src/cursor/cur_version.c

```diff
@@ -0,0 +1,123 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#include "wt_internal.h"
+
+/*
+ * __curversion_get_key --
+ *     WT_CURSOR->get_key implementation for version cursors.
+ */
+static int
+__curversion_get_key(WT_CURSOR *cursor, ...)
+{
+    WT_UNUSED(cursor);
+    return (0);
+}
+
+/*
+ * __curversion_get_value --
+ *     WT_CURSOR->get_value implementation for version cursors.
+ */
+static int
+__curversion_get_value(WT_CURSOR *cursor, ...)
+{
+    WT_UNUSED(cursor);
+    return (0);
+}
+
+/*
+ * __curversion_set_key --
+ *     WT_CURSOR->set_key implementation for version cursors.
+ */
+static void
+__curversion_set_key(WT_CURSOR *cursor, ...)
+{
+    WT_UNUSED(cursor);
+}
+
+/*
+ * __curversion_next --
+ *     WT_CURSOR->next method for version cursors.
+ */
+static int
+__curversion_next(WT_CURSOR *cursor)
```

Comment:
From what I understand in the ticket, the next() method is expected to get the next version of the current key, but would this be confusing since with the standard cursor we usually expect the next() method to go to the next key?

Reply:
Yes, we need to document this.